tests/test_attribute_selector_block.py

```diff
@@ -0,0 +1,51 @@
+from nio.block.terminals import DEFAULT_TERMINAL
+from nio.signal.base import Signal
+from nio.testing.block_test_case import NIOBlockTestCase
+from ..attribute_selector_block import AttributeSelector
+
+
+class TestExample(NIOBlockTestCase):
+
+    def test_blacklist_signals(self):
+        """The 'hello' signal attribute will get blacklisted"""
+        blk = AttributeSelector()
+        self.configure_block(blk, {'specify_behavior': 'BLACKLIST',
+                                   'specify_attributes': [{'item': 'hello'}]})
+        blk.start()
+        blk.process_signals([Signal({"hello": "n.io"})])
+        blk.stop()
+        self.assert_num_signals_notified(1)
+
+        # should have blacklisted the one incoming signal attribute
+        self.assertDictEqual(
+            self.last_notified[DEFAULT_TERMINAL][0].to_dict(),
+            {})
+
+    def test_whitelist_signals(self):
+        """'hello' attribute passes through the block unmodified."""
+        blk = AttributeSelector()
+        self.configure_block(blk, {'specify_behavior': 'WHITELIST',
+                                   'specify_attributes': [{'item': 'hello'}]})
```

Comment:
Same (but opposite) comment as previous test. Add some attributes that should be removed